Patch to fix regression: query parsing in local CGI URIs.  Based on
upstream commit e96665134ae9b5106fe0d5c30d445212d970e91b

diff --git a/src/protocol/uri.c b/src/protocol/uri.c
index 5130ec2..0d14972 100644
--- a/src/protocol/uri.c
+++ b/src/protocol/uri.c
@@ -143,7 +143,7 @@ static int
 check_whether_file_exists(unsigned char *name)
 {
 	/* Check POST_CHAR etc ... */
-	static const unsigned char chars[] = POST_CHAR_S "#?";
+	static const unsigned char chars[] = POST_CHAR_S "#";
 	int i;
 	int namelen = strlen(name);
 
